transportation_protocols/tcp.py

```diff
@@ -0,0 +1,35 @@
+import socket
+
+
+class TCP_Connection:
+
+    local_ip = "127.0.0.1"
+    
+    def __init__(self, destination_port, is_client, destination_ip=local_ip):
```

Comment:
What if we want a socket that is both a server and a client?

Reply:
this, and the parameters: initiator_ip, initiator_port, responder_ip, responder_port, timeout, and buffer size.